engine/src/main/java/com/datatorrent/stram/plan/physical/PhysicalPlan.java

```diff
@@ -1892,4 +1947,22 @@ public Integer getStreamCodecIdentifier(StreamCodec<?> streamCodecInfo)
       return null;
     }
   }
+
+  /**
+   * A interface object for the statsListener to access the DAG, it provides methods
```

Comment:
An interface for the {@link StatsListener} ... Please fix spelling and formatting of other Javadoc comments as well.